api/src/main/java/io/github/eckig/grapheditor/window/PanningWindow.java

```diff
@@ -140,30 +140,29 @@ public void setEditorProperties(final GraphEditorProperties pEditorProperties)
      */
     public void panTo(final double x, final double y)
     {
-        if (canNotPan())
-        {
-            return;
-        }
-
-        final double newX = checkContentX(x);
-        final double newY = checkContentY(y);
-        if (newX != getContentX() || newY != getContentY())
-        {
-            contentX.set(newX);
-            contentY.set(newY);
-        }
+        panToX(x);
+		panToY(y)
```

Comment:
Maybe a `;`

Reply:
Arf no I can't test directly my env is set for my works so I can't change this. I just copy/past my change in web editor to apply the change.

Sorry....